presto-main/src/main/resources/webapp/assets/presto.css

```diff
@@ -0,0 +1,551 @@
+    border: none;
+    border-top: 2px solid #454A58;
+    border-bottom: 3px solid #454A58;
+}
+
+.toolbar-row .input-group .form-control {
+    margin-top: 2px;
+    margin-bottom: -2px;
+    width:98%;
+}
+
+.toolbar-row .dropdown-menu li a {
+    color: #424242;
+}
+
+.toolbar-row .dropdown-menu li .selected {
+    background-color: #418194;
+    color: #fff;
+    font-weight: 500;
+}
+
+/** Loading indicators **/
+/** ================== **/
+
+.sparkline .loader {
+    margin: 0 0 0 90px;
+}
+
+.sparkline .loader, .loader:after {
+    width: 50px;
+    height: 50px;
+}
+
+/** ================== **/
+/** Query Details Page **/
+/** ================== **/
+
+#query-detail {
+    color: #cecece;
+    font-size: 16px;
+}
+
+.query-links {
+    text-align: right;
+    margin-top: 20px;
+}
+
+.progress-large {
+    margin: 0;
+    background-color: #808286;
+    height: 36px;
+    width: 100%;
+}
+
+.progress-large .progress-bar {
+    font-size: 18px;
+    line-height: 30px;
+}
+
+#query-detail pre {
+    border: 1px #fff solid;
+}
+
+/** Summary Section **/
+/** =============== **/
+
+.info-title {
+    width: 170px;
+    font-weight: 500;
+    text-align: right;
+}
+
+.info-text {
+    color: #fff;
+}
+
+.table > tbody > tr > .info-sparkline-text {
+    color: #fff;
+    border-top: none;
+    text-align: right;
+    padding-top: 0;
+    font-size: 23px;
+}
+
+.stack-trace {
+    font-size: 10px;
+    line-height: 14px;
+    word-wrap: normal;
+    overflow-y: hidden;
+}
+
+/** Stage List **/
+/** =========== **/
+
+#stage-list{
+    margin-top: 5px;
+    text-align: right;
+}
+
+.stage-id {
+    font-size:72px;
+    color: #b5b5b5;
+    text-align: center;
+}
+
+.single-stage-table > tbody > tr > td {
+    border-top: none;
+}
+
+.stage-table .stage-table-stat-header {
+    color: #fff;
+}
+
+.stage-table {
+    width: 170px;
+    display: inline-table;
+    margin: -8px;
+}
+
+.stage-table-time {
+    width: 170px;
+}
+
+.stage-table-memory {
+    width: 140px;
+}
+
+.stage-table-tasks {
+    width: 100px;
+}
+
+.stage-table .stage-table-chart-header {
+    color: #fff;
+    text-align: center;
+}
+
+.stage-table tr td {
+    padding-right: 5px;
+}
+
+.stage-table tr th {
+    padding-right: 5px;
+}
+
+.stage-table-stat-title {
+    font-size: 12px;
+    line-height: 16px;
+    text-align: right;
+    text-transform: uppercase;
+    font-style: italic;
+    color: #757575;
+    width: 80px;
+}
+
+.stage-table-stat-text {
+    text-align: left;
+}
+
+.bar-chart-container {
+    padding-top: 10px;
+}
+
+.histogram-table {
+    margin-right: 10px;
+}
+
+.histogram-container {
+    padding-top: 10px;
+}
+
+.table > tbody > tr > td .expand-charts-container {
+    padding-top: 40px;
+}
+
+.expand-charts-button {
+    padding: 34px 3px 34px 4px;
+    background-color: #3A3F4C;
+}
+
+.expanded-chart-title {
+    width: 100px;
+    padding-right: 5px;
+}
+
+.expanded-chart {
+    display: inline-table;
+}
+
+/**
+The MIT License (MIT)
```

Comment:
Why is this here? If some of the code is copied from somewhere else, I would move it to a separate file